pyutil, daemon: let python script both compatible with python2 and python3

holishing opened the pull request:
Python 2 [EOL date](https://www.python.org/dev/peps/pep-0373) is on Jan, 2020.

Though patches are not fully tested(only let most of them can be excuted), still open this PR.
Hope further fixes can be figured out, and python version issue in this repo can be solved,

Others:
* fix detail of PR #44
* refine indentation, convert TAB to 8 spaces.

holishing committed:
util/pyutil/big5_gen.py: remove the space between 'print' and '('

let them looks more like a function call.
fix detail of PR #44

hungte commented on daemon/angelbeats/angel_perf.py:
```diff
@@ -34,12 +37,12 @@
               10:32 C �W�u�F (�٤��έp - �Q����~��@��)
               10:33 A �אּ���� (�٤��έp - �Q����~��@��)
               10:40 �t�ΦA���i��έp�A���O��p�U:
-	            (ABC���b�u�W�AA�����AB��AC�}�I�s)
```
Please move tab/space changes to a new change. That is not related to python 2/3.

hungte commented on util/pyutil/big5.py:
```diff
@@ -3,7 +3,7 @@
 from builtins import bytes, chr
 
-import big5_tbl
+from . import big5_tbl
```
please rephrase your commit message as

Use explicit relative import for python3

hungte commented on util/pyutil/pttpost.py:
```diff
@@ -4,6 +4,9 @@
 # with non-standard Big5 data (ex, Big5-UAO or escape sequence between DBCS).
 
+from __future__ import print_function
+from builtins import zip,map
```
zip, map

hungte commented on util/pyutil/big5.py:
```diff
@@ -1,30 +1,33 @@
+    ret = ''
     b = 0
     for i in s:
-	if b:
```
please move space/tab changes to a new commit.

hungte commented on util/pyutil/pttstruct.py:
```diff
@@ -1,8 +1,12 @@
 # -*- coding: Big5 -*-
 
+from __future__ import print_function
+from builtins import zip,map
```
zip, map

hungte commented on daemon/angelbeats/angel_perf.py:
```diff
@@ -1,6 +1,9 @@
 #-*- coding: big5 -*-
 
+from __future__ import print_function,unicode_literals
+from builtins import bytes,chr,map
```
bytes, chr, map

hungte reviewed (changes requested):
In general I'm fine with the change, but please:

- move all space/tab changes to a new commit, not having them in the 2/3 compatible changes.
- follow same coding style for imports, e.g. 'import zip, map' instead of 'import zip,map'

hungte commented:
Meanwhile, how did you test these patches, especially the angelbeats daemon?

holishing committed:
docs: update build dependencies, commands for building

holishing committed:
pyutil: add auto-generated big5_tbl.py to gitignore

holishing committed:
pyutil: refine indentations of python scripts

holishing committed:
pyutil: let python script both compatible with python2 and python3

holishing committed:
pyutil: Use explicit relative import for python3

holishing committed:
daemon: refine indentations of python scripts

holishing committed:
daemon: let python script both compatible with python2 and python3

try to adapt related syntax format to let python scripts be executed by both python 2 or 3
